website2/docs/topology-development-topology-api-python.md

```diff
@@ -181,8 +181,7 @@ $ heron submit local \
 Note the `-` in this submission command. If you define a topology by subclassing `TopologyBuilder` you do not need to instruct Heron where your main method is located.
 
 > #### Example topologies buildable as PEXs
```

Comment:
this and `website2/docs/topology-development-streamlet-api.md` don't look DRY, they appear to have identical content given this small diff context

Reply:
There were duplicates.  They've been removed.